contrib/mod_sftp_sql.c

```diff
@@ -465,23 +465,29 @@ static int sqlstore_verify_key_raw(pool *p, struct sqlstore_data *store_data,
       store_data->select_query, strerror(errno));
 
   } else if (res == FALSE) {
-    (void) pr_log_writefile(sftp_logfd, MOD_SFTP_SQL_VERSION,
+    pr_trace_msg(trace_channel, 3,
```

Comment:
These "key does not match" log messages might be a bit noisy for `SFTPLog`, so I've moved them to trace logging.